client/src/components/forms/ResetPasswordForm.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { reduxForm, Field } from 'redux-form'
+import _ from 'lodash'
+
+import { resetNewPassword } from '../../actions'
+
+import { resetPasswordFields } from './fieldsData'
+import AuthFormField from './formFields/AuthFormField'
+import validateEmail from '../../helpers/validateEmail'
+
+import Button from '../styles/Button'
+
+class ResetPasswordForm extends Component {
+  onFormSubmit = values => {
+    const token = this.props.resetPasswordToken
```

Comment:
use object destruction syntax for those two props values

Reply:
Done!